### PR TITLE
Feature/vih 4493 signout button is not visible on mobile

### PR DESCRIPTION
**Pull request by kkodalikk**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/VIH-4493

### Change description ###

Fix for Signout button issue in mobile

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html [hunk 1]
```diff
@@ -3,21 +3,27 @@
...
-      <ul id="navigation" class="govuk-header__navigation right-align" aria-label="Top Level Navigation">
-        <li *ngIf="loggedIn" class="govuk-header__navigation-item">
-          <a id="header-logout-link" class="govuk-header__link" href="javascript:void()" (click)="logout()" i18n="@@header_a">
-            Sign out
-          </a>
-        </li>
-      </ul>
-    </nav>
+    <div>
+      <button type="button" role="button" class="govuk-header__menu-button js-header-toggle" aria-controls="navigation"
```
Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html]: Put in the `i18n` here as well?

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html [hunk 2]
```diff
@@ -3,21 +3,27 @@
...
+        aria-label="Show or hide Top Level Navigation" aria-expanded="false" (click)="logout()">Sign out</button>
```
Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html]: Check if you need the `aria-label` here and what content it should be, it looks like this current text isn't quite right:

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute

File: ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html (hunk 1 above)
Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html]: for `role="button"` confirm if that's needed or if the `<button>` tag is sufficiant:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Comment (ddikman) [inline ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html]: Check the `aria-controls`:
https://www.heydonworks.com/article/aria-controls-is-poop

Comment (RuwanikaWeerakkody) [reply ServiceWebsite/ServiceWebsite/ClientApp/src/app/modules/shared/header/header.component.html]: <button> tag is sufficient. The role is for specifying other elements that behave like a button.

**Pull request merged**